	Taken from OpenWrt, this patch fixes for a broken daemon() function
	in applications linked against pthread, as the fork() function provided
	by pthread has to be used then. It is said, that this won't fix for
	statically compiled binaries.

	See here: https://dev.openwrt.org/attachment/ticket/4809/fix-daemon-pthread.patch
	and here: https://bugs.busybox.net/show_bug.cgi?id=193
--- a/libc/unistd/daemon.c	2006-07-05 21:52:39.000000000 +0200
+++ b/libc/unistd/daemon.c	2009-03-21 08:45:25.000000000 +0100
@@ -54,7 +54,7 @@
 libc_hidden_proto(dup2)
 libc_hidden_proto(setsid)
 libc_hidden_proto(chdir)
-libc_hidden_proto(fork)
+/* libc_hidden_proto(fork) */
 
 int daemon( int nochdir, int noclose )
 {
